### DISCUSSION TITLE
Add union support

### DISCUSSION THREAD

**Discussion by dsgallups** [SeaQL/sea-orm#2118] [Ideas]
It'd be really cool to be able to `union` two tables together and then `into_model<T>() where T: FromQueryResult`, is there any other interest in this feature? If so, I'd be happy to contribute to this area

**Comment by tyt2y3** [comment]
Yeah, that sounds interesting. But can you illustrate how `into_model` should work? Do you mean we automatically generate a struct with columns from both tables?
